edge-modules/edgehub-proxy/Cargo.toml

```diff
@@ -7,17 +7,18 @@ edition = "2018"
 [dependencies]
 clap = "2.31"
 env_logger = "0.5"
-failure = "0.1"
-failure_derive = "0.1"
-hyper = "0.12.36"
+hyper = "0.14"
+thiserror = "1"
 log = "0.4"
 serde_json = "1.0"
-tokio = "0.1"
+tokio = { version = "1", features = ["full"] }
```

Comment:
Most likely, we don't need "full". Please change it to the set of features actually needed.

Reply:
Addressed above in Roberts comment